data/.eslintrc.yml

```diff
@@ -1,10 +1,5 @@
-extends: eslint:recommended
-
 env:
   worker: true
-  node: false
-  es6: true
-  browser: true
```

Comment:
Really?  this TURNS OFF node in the data dir.   the data dir should be 'webby' not 'node' y.   I don't recall which objects were problems here, but it was something, which is why I did it :)